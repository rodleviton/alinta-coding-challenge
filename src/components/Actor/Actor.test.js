import React from 'react';
import ReactTestRenderer from 'react-test-renderer';
import Movie from './';

describe('Actor Component', () => {
  it('should render correctly', () => {
    const props = {
      name: 'Richard Dreyfuss',
      roles:[
        {
          name: 'The Writer',
          actor: 'Richard Dreyfuss',
          movie: 'Stand By Me'
        }
      ]
    };

    const tree = ReactTestRenderer.create(
      <Movie actor={props} />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
