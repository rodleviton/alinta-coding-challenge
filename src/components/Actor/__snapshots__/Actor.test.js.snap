// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Actor Component should render correctly 1`] = `
<article
  className={undefined}
>
  <header
    className={undefined}
  >
    <h3>
      Richard Dreyfuss
    </h3>
  </header>
  <footer
    className={undefined}
  >
    <h4>
      Roles
    </h4>
    <p>
      The Writer
       (
      Stand By Me
      )
    </p>
  </footer>
</article>
`;
